### PR TITLE
Show a friendly message to non superusers

### PR DESCRIPTION
**Pull request by andyhd**
## What

* Catch API 403 error on login and set flag on user session to show a friendly error message instead of a 500 error page

## How to review

1. Set your user attribute `is_superuser` to `False` in your database

    ```sh
    docker-compose exec api python3 manage.py shell
    ```
    then
    ```python
    from control_panel_api.models import User
    me = User.objects.get(username='my username')
    me.is_superuser = False
    me.save()
    exit()
    ```
2. Login to the frontend
3. See a friendly error message

### CODE REVIEW

File: app/templates/home.html [hunk 1]
```diff
@@ -4,8 +4,10 @@
 
 {% block main_column %}
 
-  {% if error %}
-    <p>{{ error }}</p>
+  {% if not current_user.is_superuser %}
+  <h2 class="heading-medium">No access yet</h2>
```
Comment (xoen) [inline app/templates/home.html]: As discussed in another PR, I always prefer to indent in the templates to make them more readable and avoid confusion/bugs.

File: app/templates/home.html [hunk 2]
```diff
@@ -4,8 +4,10 @@
 
 {% block main_column %}
 
-  {% if error %}
-    <p>{{ error }}</p>
+  {% if not current_user.is_superuser %}
```
Comment (xoen) [inline app/templates/home.html]: Not sure I'm keen on this approach. It works but it's a bit strange that the UI determine if the user is a super user, and also we set this flag to show a permission error message.

I'm thinking, maybe we can set something like `req.errors` or something like that.

File: app/templates/home.html (hunk 1 above)
Comment (andyhd) [reply app/templates/home.html]: Yep, I knew you would say that - I will change

File: app/templates/home.html (hunk 2 above)
Comment (andyhd) [reply app/templates/home.html]: This is just a stop-gap solution while we wait for proper authorization in the API. Then we'll get the permissions from the API and check those instead.

Comment (xoen) [reply app/templates/home.html]: 👍

**Pull request merged**